javaparser-core/src/main/java/com/github/javaparser/ast/body/MethodDeclaration.java

```diff
@@ -21,10 +21,7 @@
  
 package com.github.javaparser.ast.body;
 
-import com.github.javaparser.ast.AccessSpecifier;
-import com.github.javaparser.ast.DocumentableNode;
-import com.github.javaparser.ast.NamedNode;
-import com.github.javaparser.ast.TypeParameter;
+import com.github.javaparser.ast.*;
```

Comment:
Hopefully this was correct. I'm using the JavaParser code style for idea from dev-files.

Might be a good idea to add a .editorconfig? Some files have tabs, some spaces.